### ISSUE TITLE
[Bug Report] SDL2 Update change mapping controllers on Linux

### ISSUE THREAD

**Issue by liberodark** [yuzu-emu/yuzu#9399]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3194 / Mainline 1262

### Description of Issue

Hi,

I'm opening this issue to follow up on a bug already reported on Discord.
Edit* Since PR #9374 was merge this *change the guid of controllers support under Linux / Steam Deck.

Best Regards

### Expected Behavior

Controllers isn't work on Linux env

### Reproduction Steps

Run Yuzu on Linux or Steam Deck with a controller

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10170467/yuzu_log.txt)

### System Configuration

CPU: Intel Core i9-9900KF
GPU/Driver: Radeon RX 5500XT / Mesa 22.2.4
RAM: 32 Go
OS: Manjaro / Batocera / SteamOS

**Comment by german77** [comment]
Is not broken. SDL Changed the GUID of the controllers. You need to remap them. A fix will be made in the next days

**Comment by liberodark** [comment]
i know, sorry my english is not very good have update message.

**Comment by gustakasn0v** [comment]
I had this weird behaviour on Steam Deck where opening the button remapping settings and trying to remap the A and B keys would close the settings, making remapping impossible. I tried this on desktop mode, with Steam open (ie. having the Steam Virtual Input device) and also closed.

I was able to get around this by adding Yuzu as a non-Steam game, and opening it on Steam mode. The issue still happens intermittently but I was able to remap. Attempting to remap after holding the steam button to use the right trackpad as mouse seemed to always fail; I had more success touching the screen to navigate
